chapter_linear-networks/linear-regression_vn.md

```diff
@@ -391,7 +394,10 @@ Note that even if our function is truly linear and noiseless, these parameters w
 although the algorithm converges slowly towards a local minimum it cannot achieve it exactly in a finite number of steps.
 -->
 
-*dịch đoạn phía trên*
+Sau khi huấn luyện bằng một số lần lặp được xác định trước (hoặc cho đến khi đạt được một số tiêu chí dừng nhất định),
+chúng ta sẽ ghi nhận các tham số mà mô hình ước tính, được ký hiệu là $\hat{\mathbf{w}}, \hat{b}$ (một cách thống nhất thì biểu tượng "mũ phía trên" biểu thị ước tính).
+Lưu ý rằng ngay cả khi chức năng của chúng ta thực sự tuyến tính và không có hiễu, các tham số này sẽ không giảm thiểu chính xác của mất mát vì,
```

Comment:
Đừng quá lạm dụng google dịch :)) bạn nên đọc tí toán bằng tiếng Anh đi nhé
```suggestion
Lưu ý rằng ngay cả khi hàm số thực sự tuyến tính và không có nhiễu, các tham số này sẽ không tối thiểu hóa được hàm mất mát.
```

Reply:
À, đoạn này em chỉnh mà bị bỏ sót :)))